source/Sashimi/Sashimi.csproj

```diff
@@ -26,7 +26,7 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Sashimi.AzureScripting" Version="9.0.2" />
-    <PackageReference Include="Sashimi.Server.Contracts" Version="8.3.4" />
+    <PackageReference Include="Sashimi.AzureScripting" Version="9.1.0" />
+    <PackageReference Include="Sashimi.Server.Contracts" Version="8.5.4" />
```

Comment:
Do we need to bump contracts here? Azure scripting still uses `8.3.4` https://github.com/OctopusDeploy/Sashimi.AzureScripting/blob/main/source/Sashimi/Sashimi.csproj

Reply:
I am just working on a PR to update Sashimi.AzureScripting and then I will reference the new version here

Reply:
I have updated this version now